work-cards.js

```diff
@@ -0,0 +1,99 @@
+const workSec = document.getElementById('work-sec');
+const div = document.createElement('div');
+const cardInfo = [{
+  link: '/index.html',
+}];
+
+workSec.appendChild(div);
+
+div.innerHTML = `<div class="works">
+<div class="project-1 p1">
+    <img class="project-1-img" src="res/img_Placeholder.png" alt="Project-1">
+    <div class="project1-txt">
+        <h3 class="project-1-h3">Multi-Post Stories</h3>
+        <p class="project-1-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the .</p>
+    </div>
+    <div class="siklls">
+        <ul class="skills-ul">
+            <li><a href="${cardInfo[0].link}">css</a></li>
+            <li><a a href="${cardInfo[0].link}">html</a></li>
+            <li><a a href="${cardInfo[0].link}">bootstrap</a></li>
+            <li><a a href="${cardInfo[0].link}">Ruby</a></li>
+        </ul>
```

Comment:
- [ ] **Required:** Kindly update your styles to match the template design.

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/198221368-c5828507-6961-4585-8a32-a31c010f43d7.png) | ![image](https://user-images.githubusercontent.com/27709832/198221442-f8f92ab3-f134-4de4-a804-6234734e484f.png)

1. Kindly increase the size of the header or increase its weight.
2. Move the language tags to align to the left as the header above it and the image beneath.

- [ ] **Optional:**

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/198222450-e14e4419-efcd-4762-ba20-f5e18da2eae5.png) | ![image](https://user-images.githubusercontent.com/27709832/198222531-e75648ff-75c2-4de1-bc8a-3ea325eb4247.png)

1. I personally think the buttons are too tall. Kindly reduce the margin/padding-top and bottom.
2. Also, reduce the size of the icons.
3. Increase the font size.